### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_Reflow: In Reflow resolution “Menu hamburger” button is not accessible by “Enter” & “Space” key 

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#177]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:

#A11yTCS;#A11yMAS;#A11ySev3;#30days;#P3_30days_Dec22;#A11yWCAG2.1;#WCAG1.4.10;#DesktopWeb;#Win11;#ChromiumEdge
### Environment Details:
Application Name: 30days 
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Pre-Requisite:
To apply Reflow resolution, follow the below steps:
1.	Open DevTools in new window by press F12.
2.	Adjust the application browser window size to 320*256 pixels using mouse (Change the browser zoom levels accordingly up to 200% to adjust the window pixels to 320*256).
3.	Refresh the page.

### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/ 
2.	Home page will open then tab to "Menu Hamburger" button present in header section.
3.	Observe in Reflow resolution “Menu Hamburger” button is not accessible by “Enter” & “Space” key
### Actual:
In Reflow resolution “Menu Hamburger” button is not accessible by “Enter” & “Space” key.
Note – This issue is also observed in 200% resize
### Expected:
In Reflow resolution “Menu Hamburger” button should be accessible by “Enter” & “Space” key.
### User Impact:
Low vision user will face difficulty while accessing “Menu Hamburger” button.
### Attachment:
[209560677-1ca72026-c6bb-4719-96b4-1dde0ff81457.webm](https://user-images.githubusercontent.com/85863895/211751465-5321d506-8fb6-4b5a-b7df-1bfd4ca056b0.webm)

**Comment by nitya** [comment]
@Ashutoshsahoo164  I tried this on a macOS machine with Microsoft Edge and the Space button worked (if held down, the menu slides over) but not Enter. Do both need to be active?

**Comment by Ashutoshsahoo164** [comment]
Hi @nitya Please try on Windows Machine . We need any one of them to be worked either "Enter" or "Space"

**Comment by Ashutoshsahoo164** [comment]
As per the above comments , closing this bug.
